### DISCUSSION TITLE
Salt failure alerts possible?

### DISCUSSION THREAD

**Discussion by csanate** [Security-Onion-Solutions/securityonion#7585] [Unsupported Versions] [answered]
Hello, I think we are finally getting a handle on salt, but one recurring issue I seem to keep running into is when the global.sls file gets updated incorrectly, causing a salt render error. Say for example a threshold or bpf is added incorrectly. 

Is there any way to get elastalert or some other mechanism to alert when salt is failing to run? Without invoking salt directly, we never seem to know that there is an issue until later. 

Thanks!

**Comment by dougburks** [comment, marked as answer]
You could potentially develop a cron job that checks for salt render errors and then sends some kind of alert into Elasticsearch. However, it may be just as easy (and more immediate) if you just run `sudo salt-call state.highstate` after your change.
